### PR TITLE
feat: Add Nginx support

### PR DESCRIPTION
**Pull request by anurag-rajawat**
# Description
This PR 
- Adds nginx njs script to observe API calls.
- Adds support for [Nginx Inc](https://github.com/nginxinc/kubernetes-ingress/) ingress controller.
- Adds support to observe API calls made to an application backed by nginx web server on a VM.

### CODE REVIEW

File: sentryflow/config/default.yaml
Comment (PrimalPimmy) [inline sentryflow/config/default.yaml:1]: so we are commenting both configs (istio sidecar and njs) and letting the users decide? Would it be better to use just two separate config files for each of these? Or make it configurable elsewhere?

**Review by PrimalPimmy — COMMENTED** [review]
Some questions

File: sentryflow/config/default.yaml
Comment (anurag-rajawat) [reply sentryflow/config/default.yaml:1]: > Would it be better to use just two separate config files for each of these? Or make it configurable elsewhere?

If we have `n` integration sources, we'll need to maintain config files for each of those `n` sources. That's why we have chosen to use a [slice](https://github.com/5GSEC/SentryFlow/blob/main/sentryflow/pkg/config/config.go#L36)  for the receivers.

**Review by daemon1024 — COMMENTED** [review]
LGTM 👌

**Review by daemon1024 — APPROVED** [review]

**Review by PrimalPimmy — APPROVED** [review]
LGTM

**Pull request merged**